.spi.yml

```diff
@@ -5,9 +5,8 @@ builder:
     scheme: ComposableArchitecture
   - platform: macos-xcodebuild
     scheme: ComposableArchitecture
-  - platform: macos-xcodebuild-arm
-    scheme: ComposableArchitecture
   - platform: tvos
     scheme: ComposableArchitecture
   - platform: watchos
     scheme: ComposableArchitecture_watchOS
+  - documentation_targets: [ComposableArchitecture]
```

Comment:
This sets the list of documentation targets for all configs (but we're only actually building docs for `macos-spm` / `5.6` unless an explicit platform and/or Swift version is configured.)